platform/android/java/src/org/godotengine/godot/Godot.java

```diff
@@ -338,6 +341,12 @@ public static Godot getInstance() {
 		return Godot._self;
 	}
 
+	public int getGLESVersionCode() {
+		ActivityManager am = (ActivityManager) Godot.getInstance().getSystemService(Context.ACTIVITY_SERVICE);
```

Comment:
`clang-format` has a small nitpick on the cast here: https://travis-ci.org/godotengine/godot/jobs/396440442